aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -32,6 +38,28 @@ Tensor& fill_(Tensor& self, const Tensor& value) {
   return self._fill_(value);
 }
 
+Tensor mvlgamma(const Tensor& self, int64_t p) {
+  AT_CHECK(at::isFloatingType(self.type().scalarType()),
+           "mvlgamma is not implemented for ", self.type());
+  AT_CHECK((self > 0.5 * (p - 1.)).all().toCByte(),
+           "Condition for computing multivariate log-gamma not met");
+  AT_CHECK(p >= 1, "p has to greater than or equal to 1");
```

Comment:
has to be

Reply:
Oops, sorry. Has been fixed the following commit. Thank you.